src/main/java/org/jabref/gui/autocompleter/AutoCompletePreferences.java

```diff
@@ -13,15 +13,15 @@
     private AutoCompleteFirstNameMode firstNameMode;
     private boolean onlyCompleteLastFirst;
     private boolean onlyCompleteFirstLast;
-    private List<String> completeNames;
-    private JournalAbbreviationPreferences journalAbbreviationPreferences;
+    private List<String> completeFields;
+    private final JournalAbbreviationPreferences journalAbbreviationPreferences;
 
     public AutoCompletePreferences(boolean shouldAutoComplete, AutoCompleteFirstNameMode firstNameMode, boolean onlyCompleteLastFirst, boolean onlyCompleteFirstLast, List<String> completeNames, JournalAbbreviationPreferences journalAbbreviationPreferences) {
```

Comment:
Nitpick: If you rename this to `completeFields` then can you please also rename the input parameter if the constructor?